adoc/extensions/index.adoc

```diff
@@ -10,4 +10,6 @@ specification, but their design is subject to change.
 // leveloffset=2 allows extensions to be written as standalone documents
 // include::sycl_khr_extension_name.adoc[leveloffset=2]
 
+
 include::sycl_khr_default_context.adoc[leveloffset=2]
+include::sycl_khr_free_function_commands.adoc[leveloffset=2]
```

Comment:
```suggestion
include::sycl_khr_default_context.adoc[leveloffset=2]

include::sycl_khr_free_function_commands.adoc[leveloffset=2]
```

Reply:
This is just adjusting the spacing between the sections, right?  Why is that necessary?

Reply:
Changes are needed if you want the new index item to show up in a new line.

Reply:
As editor, I'd like to understand this.  I tried removing the blank line and building the spec.  I didn't see any difference one way or the other.  I even did a "diff" of the generated html and pdf files, and I didn't see any difference.  What do you mean by "if you want the new index item to show up in a new line"?

Reply:
Interesting. looks like this is an issue with the GitHub build-in rendering only.
This is what I checked, and I assume we don't need workaround for that.

Reply:
Thanks for explaining.  No, I don't think the GitHub rendering is that important here.  GitHub doesn't even follow the `include`, so it's not very accurate anyway.

I was interested to understand this because I've been thinking of removing the blank lines between the `include` statements in the top-level `syclbase.adoc` file.